politeiawww/userwww.go

```diff
@@ -814,8 +812,10 @@ func (p *politeiawww) setCMSUserWWWRoutes() {
 		p.handleChangeUsername, permissionLogin)
 	p.addRoute(http.MethodPost, www.RouteChangePassword,
 		p.handleChangePassword, permissionLogin)
+	p.addRoute(http.MethodGet, www.RouteUserDetails,
+		p.handleCMSUserDetails, permissionLogin)
```

Comment:
Do admins need the ability to see the user details of any cms user?

Reply:
Will be making that change as discussed on matrix.